README.rst

```diff
@@ -4,6 +4,21 @@
   .. image:: https://coveralls.io/repos/broadinstitute/oncotator/badge.png?branch=issue_211_travis
     :target: https://coveralls.io/r/broadinstitute/oncotator?branch=issue_211_travis
 
+======================
+ONCOTATOR IS NO LONGER SUPPORTED OR MAINTAINED
```

Comment:
I'd go less all caps, but maybe that's me.

Oncotator has been deprecated and is no longer supported or maintained

Reply:
That's fine - I wanted to make it "pop".